docs/courses/coursemanagement.md

```diff
@@ -254,6 +254,18 @@ You can now
 ![Book Mapping](/img/bookmapping.png)
 
 ## Publishing
+
+As you update your assignment, a **Publish** button will show to the left of your username in the top menu bar.  
+
+![Quick Publish](/img/assignmentpubish.png)
+
+This can be used as a shortcut to publish dialog. There is also an option to publish in the **Education** menu
```

Comment:
Add image or more description of where the Education menu is -- we know it's in the top tool bar but the user might not